mypyc/emit.py

```diff
@@ -712,3 +713,28 @@ def emit_gc_clear(self, target: str, rtype: RType) -> None:
             self.emit_line('Py_CLEAR({});'.format(target))
         else:
             assert False, 'emit_gc_clear() not implemented for %s' % repr(rtype)
+
+
+class TupleNameVisitor(RTypeVisitor[str]):
```

Comment:
What about writing a few plain unit tests for this? This would be easy to unit test.